public/src/App.css

```diff
@@ -0,0 +1,3 @@
+.btn__content {
+  text-transform: capitalize !important;
```

Comment:
`!important` ???? who uses important

Reply:
@nicksarafa Why not in the case of overriding vuetify built-in styles?

Reply:
👍